guide.md

```diff
@@ -0,0 +1,406 @@
+Getting Started 2.0
+===
+
+For this guide, we're going to walk through some simple examples for each of the
+pieces of CRUD. Each step in this guide will build on the previous, and is meant
```

Comment:
Is it worth explaining the acronym `CRUD` here. Generally, in guides like this, it's a good idea to define each term the first time it's used.